internal/cloud/realm/access_list.go

```diff
@@ -26,6 +32,29 @@ type allowedIPCreatePayload struct {
 	UseCurrent bool   `json:"use_current,omitempty"`
 }
 
+func (c *client) AllowedIPs(groupID, appID string) (AccessList, error) {
+	res, resErr := c.do(
+		http.MethodGet,
+		fmt.Sprintf(allowedIPsPathPattern, groupID, appID),
+		api.RequestOptions{},
+	)
+	if resErr != nil {
+		return AccessList{}, resErr
+	}
+
+	if res.StatusCode != http.StatusOK {
+		return AccessList{}, api.ErrUnexpectedStatusCode{"get allowed ips", res.StatusCode}
```

Comment:
depending on where we land with naming the client method, this should update accordingly